fbw-a380x/src/systems/systems-host/systems/LegacyGpws.ts

```diff
@@ -207,8 +209,12 @@ export class LegacyGpws {
     );
   }
 
-  update(deltaTime) {
-    this.gpws(deltaTime);
+  update(deltaTime: number) {
+    const throttledT = this.updateThrottler.canUpdate(deltaTime);
+
+    if (throttledT !== 1) {
```

Comment:
Should be > 0? canUpdate returns -1 when you can't update.

Reply:
Oh shoot, I was meant to check for ``-1``
Replace it now, thanks for noticing. We're good to go now :)